docs/computer/Acoustics.md

```diff
@@ -4,67 +4,91 @@ author: Syed Jawad Akhtar
 email: syedjawadakhtar@gmail.com
 ---
 
-> :warning: **Documentation is still in progress** 
+> :warning: **WIP** 
 
 ## Acoustics Implementation - Survey, procurement & implementation
 
-#### Introduction about acoustics and competitions using it
-[Acoustics](https://en.wikipedia.org/wiki/Acoustics) is the branch of physics related to the study of mechanical waves in gases, liquids, and solids. [Underwater Acoustics](https://en.wikipedia.org/wiki/Acoustics#Underwater_acoustics) is the scientific study of natural and man-made sounds underwater. Applications include sonar to locate submarines, underwater communication by whales, et cetra. To make our AUVs of industrial level standards and perform properly among the different sounds produced underwater, various competitions all around the world oncorporate acoustic localisation tasks in there arena. For example in [Singapore AUV Challenge](https://sauvc.org/), [acoustic pingers](https://ocean-innovations.net/companies/rje-international/acoustic-pingers-and-transponders/) located at different props are used for [target acquisitions](https://sauvc.org/rulebook/#2.-target-acquisition) and [localization](https://sauvc.org/rulebook/#4.-localization)). Similarly the [Robosub competition](https://robosub.org/) conducted by AUVSI also plants diffrent frequency pingers to guide the vehicle around the arena. Check out [Resources - RoboSub](https://robosub.org/resources/) link for their configurations as they change the theme every year.
+#### Introduction 
```

Comment:
Sub-headings should go in level.
Try to give only one h1 followed by sub-headings as h2. If required then give h3 and h4.